### PR TITLE
docs: updated Installing Podman with Openshift Local prerequisites

### PR DESCRIPTION
**Pull request by themr0c**
* Added Enterprise or Professional edition requirement

Signed-off-by: Fabrice Flore-Thébault <ffloreth@redhat.com>

### CODE REVIEW

**Review by gbraad — COMMENTED** [review]
@anjannath

File: website/docs/Installation/windows-install/installing-podman-with-openshift-local.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ Consider installing the Podman Preset in OpenShift Local rather than [installing
 #### Prerequisites
 
 * The user is a member of the *Hyper-V Administrators* group. The user can add a virtual machine, start or stop this virtual machine, and access the resources related to this virtual machine.
+* Windows 10 or 11, Enterprise or Professional edition
```
Comment (anjannath) [inline website/docs/Installation/windows-install/installing-podman-with-openshift-local.md]: should we mention the exact version for windows 10 along with the edition (professional or enterprise)? in the CRC docs its mentioned as `Windows 10 Fall Creators Update (version 1709) or later`

ref: https://crc.dev/crc/#_microsoft_windows

Comment (themr0c) [inline website/docs/Installation/windows-install/installing-podman-with-openshift-local.md]:
```suggestion
* Windows 10 version 1709 or later, Enterprise or Professional edition, or Windows 11, Enterprise or Professional edition
```

**Review by anjannath — APPROVED** [review]

**Review by gbraad — APPROVED** [review]
LGTM

**Pull request merged**